lang/en/tool_cloudmetrics.php

```diff
@@ -85,4 +85,7 @@
 
 // Error messages.
 $string['metric_not_found'] = 'Metric \'{$a}\' not found';
-
+$string['collector_failed'] = 'Metric \'{$a}\' failed at %s';
+$string['no_issues'] = 'No issues';
+$string['checkcollectorcheck'] = 'Collector Failures';
+$string['managelink'] = 'admin/settings.php?section=tool_cloudmetrics';
```

Comment:
urls don't need to be translated so this doesn't need to be a string, just use a moodle_url where it is needed

Reply:
I copied what was done in the example you sent me earlier:

https://github.com/catalyst/moodle-auth_saml2/blob/MOODLE_39_STABLE/classes/check/certificateexpiry.php

but I'll fix it.